### PR TITLE
Disable checkbox

### PR DESCRIPTION
**Pull request by murajun1978**
- Disable checkbox when encoding from markdown to html

### CODE REVIEW

**Comment by murajun1978** [comment]
@sugarshin Could you review this PR :pray:

File: src/encoding/markdownToHTML.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ renderer.listitem = (text) => {
   /* eslint-disable no-useless-escape */
   if (/^\s*\[[x ]\]\s*/.test(text)) {
     text = text // eslint-disable-line no-param-reassign
-      .replace(/^\s*\[ \]\s*/, '<input type="checkbox" /> ')
+      .replace(/^\s*\[ \]\s*/, '<input type="checkbox" disabled /> ')
       .replace(/^\s*\[x\]\s*/, '<input type="checkbox" checked /> ');
```
Comment (sugarshin) [inline src/encoding/markdownToHTML.js]: I think that should also be applied to `checked` inputs.

Comment (murajun1978) [reply src/encoding/markdownToHTML.js]: Exactly, I'll fix it :smile:

**Comment by murajun1978** [comment]
@sugarshin I fixed it. Could you review again :pray:

**Pull request merged**